source/applicability/transport_of_data.rst

```diff
@@ -21,6 +21,55 @@ system is waiting for an answer to a previously sent message it can
 can continue to send messages. The exception in the RSMP / SXL version message
 (see section :ref:`rsmpsxl-version`).
 
+RSMP connections can be established:
+
+* Between site and supervision system.
+  See :ref:`communication establishment between sites and supervision system <communication-establishment-between-sites-and-supervision-system>`.
+  The site needs to support at multiple RSMP connections to different
+  supervisors. See :ref:`Multiple supervisors <multiple-supervisors>`.
+
+* Directly between sites.
+  See :ref:`communication establishment between sites <communication-establishment-between-sites>`.
+
+.. note::
+   Implementing support for communication between sites is not required unless
+   otherwise stated.
+
+.. _multiple-supervisors:
+
+Multiple supervisors
+^^^^^^^^^^^^^^^^^^^^
+
+Each site needs to support the following:
+
+* At least three (3) RSMP connections to different supervision systems
+  simultaneously and handle concurrent communication with all of them.
+
+  For example, a traffic light controller might connect to both a traffic
+  management system and a bus priority system. Or it might connect to both a
+  traffic management system and a secondary monitoring system.
+
+* It must be possible to configure the list of supervisors as part of the
+  RSMP configuration in the site. In the configuration, supervisors are
+  identified by their IP addresses.
+
+* It must be possible to configure supervisors as read-only. A read-only
+  supervisor is not allowed to send commands, but can only request or
+  subscribe to status.
```

Comment:
maybe we should mention request for aggregated status, which i suppose should be allowed

Reply:
Yes, aggregated status is important. But alarms are probably not needed and the watchdog messages should not alter the clock

Reply:
I've updated the draft

Reply:
If the only thing a read-only supervisor is not allow to do is to send is commands, we should be explicit and say that: everything else is allowed and must be handled.

Otherwise we can consider a table like:

|type | allowed? |
|--|--|
| CommmandRequest | reject |
| StatusRequest  | ok |
| AggregatedStatusRequest | ok |